deepfence_frontend/apps/dashboard/src/features/settings/pages/ScheduledJobs.tsx

```diff
@@ -23,43 +27,77 @@ export type ActionReturnType = {
   message?: string;
   success: boolean;
 };
+type ActionData = {
```

Comment:
Why there are two types for action function return? see `ActionReturnType`above.